Qt.py

```diff
@@ -68,14 +68,20 @@ def remap(object, name, value, safe=True):
             raise AttributeError("%s != 'module': Cannot alter "
                                  "anything but modules" % object)
 
-    __remapped__.append(name)
+    elif hasattr(object, name):
+        # Keep track of modifications
+        __modified__.append(name)
+
+    if name not in __added__:
```

Comment:
This is a minor bugfix; before, members in `__added__` were also in `__remapped__`.